### ISSUE TITLE
Error in documentation for Modelica.Math.atan3?

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#629]
**Reported by stefanv on 21 Oct 2011 18:36 UTC**
The documentation for Modelica.Math.atan3 in MSL 3.1 and 3.2 states that the returned value "y is in the range: -pi < y-y0 < pi".  One of those less-thans should be a less-than-or-equal, but it's not clear to me which one (or if it differs for y0 < 0 and y0 > 0).

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/629

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 2 Jan 2013 20:55 UTC**
Fixed in 63fde4313ad3faa1b33096cd91a48bd3b31c24f4:

The documentation has been changed to "-pi < y-y0 <= pi" and the computation has been adapted:

For the default case (y0=0), exactly the result of atan2(..) is returned.

Otherwise, the formula is derived in the following way (w = atan2(..)):

```
-pi < y - y0 <= pi
-pi < w + 2*pi*N - y0 <= pi
(-pi+y0-w)/(2*pi) < N <= (pi+y0-w)/(2*pi)
-> N := integer( (pi+y0-w)/(2*pi) )
-> y = w + 2*pi*integer((pi+y0-w)/(2*pi));

```

**Comment by modelica-trac-importer** [comment]
**Changelog removed by otter on 2 Jan 2013 20:55 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by jakesson on 3 Jan 2013 09:06 UTC**
Minor syntactic error fixed in e13ad358394322d2bf01b51e9ab8a85989167c08.
